src/models/Exporter.js

```diff
@@ -6,22 +11,24 @@ const exporterOptions = {
   fileType: {
     title: 'Export as',
     type: 'dropdown',
-    choices: ['GCode (.gcode)', 'Theta Rho (.thr)']
+    choices: [gcodeTypeName, thrTypeName, svgTypeName]
   },
   post: {
     title: 'Program end code',
-    type: 'textarea'
+    type: 'textarea',
+    isVisible: (state) => { return state.fileType !== svgTypeName },
   },
   pre: {
     title: 'Program start code',
-    type: 'textarea'
+    type: 'textarea',
+    isVisible: (state) => { return state.fileType !== svgTypeName },
   },
   reverse: {
```

Comment:
Should reverse be visible when exporting SVG? I am assuming not.

Reply:
I think it could be useful. Besides making svgs for image viewing, I am intending these to be used on axidraw type machines. I assume they don't care much about the direction, but they might care some? IDK.